projects/testing-library/tests/render.spec.ts

```diff
@@ -22,15 +22,15 @@ import { render, fireEvent } from '../src/public_api';
 class FixtureComponent {}
 
 test('creates queries and events', async () => {
-  const component = await render(FixtureComponent);
+  await render(FixtureComponent);
```

Comment:
This test should also be left, as-is.
It's here to test those methods, it's just considered a bad practice to use them.
Maybe we should try to deprecate them... but for now I would prefer to under test.